vortex-api/src/main/java/mil/army/usace/hec/vortex/temporal/TemporalDataCalculator.java

```diff
@@ -0,0 +1,243 @@
+package mil.army.usace.hec.vortex.temporal;
+
+import hec.heclib.util.Heclib;
+import mil.army.usace.hec.vortex.VortexGrid;
+
+import java.time.Duration;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Logger;
+
+class TemporalDataCalculator {
+    private static final Logger logger = Logger.getLogger(TemporalDataCalculator.class.getName());
+
+    private TemporalDataCalculator() {
+        // Utility Class
+    }
+
+    public static float[] calculateWeightedAccumulation(List<VortexGrid> grids, ZonedDateTime startTime, ZonedDateTime endTime) {
```

Comment:
Can most of the methods in this class be package-private rather than being declared `public`?